cmd/core/node_association.go

```diff
@@ -76,3 +65,57 @@ func (association *NodeAssociation) ScheduleHeartbeatRequest(duration time.Durat
 	}(ctx, duration)
 	return cancel
 }
+
+func (association *NodeAssociation) HeartbeatScheduler(conn *PfcpConnection) {
+
+	association.HeartbeatsActive = true
+
+	ctx := context.Background()
+
+	var sequence uint32
+	sequence = association.NewSequenceID()
+	SendHeartbeatRequest(conn, sequence, association.Addr)
```

Comment:
Try to move `SendHeartbeatRequest` lines into for cicle. After that you can remove `SendHeartbeatRequest` from the `case seq := <-association.HeartbeatChannel` clause as well.